### ISSUE TITLE
how to see a accuracy with increasing and change a hyper parameter??

### ISSUE THREAD

**Issue by kjunhwa** [hx173149/C3D-tensorflow#73]
hi, thank you for share c3d-tensorflow!!

I try to run the code and success.
But i want to see the accuracy graph with epoch increasing. i just can see the step/accuracy graph on the tensorboard (running by train_c3d_ucf101.py and got  visual logs)

Next if i can, i want to change the hyper parameter(such like epoch,batch size, learning rate, etc....)

Lastly how can i change the models? ( sports1m_finetuning_ucf101.model -> something..)

Thanks for read my question and forgive my poor english.

Be happy!!
